### PR TITLE
F/mobile web view

### PR DESCRIPTION
**Pull request by juliangarritano**
View on mobiles is now full screen!

### CODE REVIEW

File: src/js/views/conversationView.js [hunk 1]
```diff
@@ -18,8 +21,32 @@ module.exports = Marionette.CompositeView.extend({
...
+        /* Android */
+        window.addEventListener('resize', function() {
+            isKeyboard = (window.innerHeight < initialScreenSize);
+            isLandscape = (screen.height < screen.width);
+
+            this.keyboardToggled();
+        }.bind(this), false);
+
+        /* iOS */
+        $('input').bind('focus blur', function() {
```
Comment (lemieux) [inline src/js/views/conversationView.js]: this is way to broad.. you'll target all inputs in the page

File: src/js/views/conversationView.js [hunk 2]
```diff
@@ -18,8 +21,32 @@ module.exports = Marionette.CompositeView.extend({
...
+        /* Android */
+        window.addEventListener('resize', function() {
+            isKeyboard = (window.innerHeight < initialScreenSize);
+            isLandscape = (screen.height < screen.width);
+
+            this.keyboardToggled();
+        }.bind(this), false);
+
+        /* iOS */
+        $('.message-input').bind('focus blur', function() {
```
Comment (lemieux) [inline src/js/views/conversationView.js]: use `this.$('.message-input')` to scope it to the widget. it's still is to broad and will target all `.message-input` in the page if any is present

**Comment by juliangarritano** [comment]
This can be improved for iOS

**Comment by juliangarritano** [comment]
Can this be merged?
I need a :+1: !

Note that the logo hiding is working on Android only right now (because iOS is funky)

File: src/js/views/conversationView.js [hunk 3]
```diff
@@ -37,6 +36,16 @@ module.exports = Marionette.CompositeView.extend({
         };
     },
 
+    keyboardToggled: function(isKeyboard) {
```
Comment (mspensieri) [inline src/js/views/conversationView.js]: nit: `isKeyboardShown` ?

**Comment by mspensieri** [comment]
:+1:

**Pull request merged**